src/web/sitemap.rs

```diff
@@ -31,3 +31,20 @@ pub fn robots_txt_handler(_: &mut Request) -> IronResult<Response> {
     resp.headers.set(ContentType("text/plain".parse().unwrap()));
     Ok(resp)
 }
+
+pub fn about_handler(_: &mut Request) -> IronResult<Response> {
+    let mut content = BTreeMap::new();
+
+    let conn = ctry!(connect_db());
```

Comment:
We are using our rd2d connection pool in our iron handlers, instead of creating a new connection. Getting `req` parameter and using `let conn = extension!(req, Pool);` should do the trick (similar to sitemap).